src/v2/Apps/Conversation/Components/__tests__/ConversationMessages.jest.tsx

```diff
@@ -0,0 +1,42 @@
+import { ConversationMessages } from "../ConversationMessages"
+import { mount } from "enzyme"
+import React from "react"
+
+describe("ConversationMessages", () => {
+  it("filters out empty and null messages", () => {
+    const wrapper = mount(
```

Comment:
This is not the _ideal_ way to test this component. It's not actually exercising the relay magic and relies on a pretty verbose mock. After two hours of trying, the "right" way still alludes me... extra motivation for the relay peer learning group!